provider/build.gradle

```diff
@@ -15,10 +15,14 @@ android {
         debug {
             debuggable true
             minifyEnabled false
+            buildConfigField("long", "VERSION_CODE", "${defaultConfig.versionCode}")
+            buildConfigField("String","VERSION_NAME","\"${defaultConfig.versionName}\"")
```

Comment:
is `String` meant to be Capitalized here?

Reply:
Yeah, it is. A Gradle file is written in Groovy (unless you explicitly switch the Gradle system to Kotlin-only - which is something we should do further down the track) and this here is pretty much referring to a `java.lang.String`-typed variable. Groovy was - back in the day - used as a way to have a scripting-friendly way of using Java, so they allow to use all Java types almost implicitly.

The slashes are for escaping the double quotes, it's "one of those things".

https://developer.android.com/reference/tools/gradle-api/4.2/com/android/build/api/variant/BuildConfigField#value